templates/cisco_asa_show_interface.textfsm

```diff
@@ -25,12 +25,14 @@ Value FIVEMIN_DROP_RATE (\d+)
 
 Start
   ^Interface.* -> Continue.Record
-  ^.*Interface ${INTERFACE} "${INTERFACE_ZONE}", is ${LINK_STATUS}.*protocol is ${PROTOCOL_STATUS}
```

Comment:
Can you remove all the static whitespaces in here and replace with `\s+` since we're expecting at least one space and it'll catch if they add more for whatever reason in a newer release.

Reply:
Didn't want to adjust the original template that was out there. WIll get the updates in.